tests/rptest/remote_scripts/cloud/pod_log_extract.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
Would it be a good idea to add `set -eu`?

Reply:
Yep.